build-config/resources/Ballerina.toml

```diff
@@ -15,6 +15,9 @@ path = "../native/build/libs/websocket-native-@project.version@.jar"
 [[platform.java11.dependency]]
 path = "./lib/http-native-@stdlib.httpnative.version@.jar"
 
+[[platform.java11.dependency]]
+path = "./lib/constraint-native-@stdlib.constraintnative.version@.jar"
```

Comment:
we need to add org, repo as well right?

Reply:
I will update with the migration PR https://github.com/ballerina-platform/module-ballerina-websocket/pull/697